mod-source-record-manager-server/src/main/java/org/folio/services/JobExecutionService.java

```diff
@@ -114,4 +115,13 @@ public interface JobExecutionService {
    */
   Future<Boolean> completeJobExecutionWithError(String jobExecutionId, OkapiConnectionParams params);
 
+
+  /**
+   *
+   * @param ids JobExecution Logs to be deleted using Ids
```

Comment:
I think "job execution logs" are referring to something different from "job executions". I think this should be `JobExecutions to be deleted using Ids`

Reply:
Hi @okolawole-ebsco - I have update the comments as per your suggestions, Thanks 🙂